### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by manosim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,38 +25,39 @@ Then you have to choose between the different configs, depending on the project
   - react
   - react-native
 
-You'll also need to include eslint & any other plugins/parsers that are needed in your package.json.
-Particularly:
+You'll also need to include eslint & any other plugins/parsers that are needed in your package.json. Particularly:
```
Comment (JakeSidSmith) [inline README.md]: Not necessary in node 5+. Perhaps this line should say that it should be a dependency only if node 4 or less.

Comment (manosim) [inline README.md]: But if you want to use the `react` you will also install the `eslint-plugin-react-native` correct?

Comment (JakeSidSmith) [inline README.md]: If they are dependencies of our config, there's no need to add them as dependencies of your project (in node 5+).

File: package.json [hunk 2]
```diff
@@ -30,9 +30,9 @@
   "dependencies": {
   },
   "devDependencies": {
-    "eslint": "=3.10.0"
-  },
-  "peerDependencies": {
-    "eslint": "=3.10.0"
+    "babel-eslint": "=7.1.0",
```
Comment (JakeSidSmith) [inline package.json]: > Note: You don't need to use babel-eslint if you are using ES2015 (ES6), ES2016 (ES7) or ES2017 (ES8). ESLint actually supports ES2015/ES2016/ES2017, JSX, and object rest/spread by default now.

https://github.com/babel/babel-eslint

**Pull request merged**